scipy/signal/_signaltools.py

```diff
@@ -2142,18 +2142,18 @@ def deconvolve(signal, divisor):
 
     Parameters
     ----------
-    signal : array_like
-        Signal data, typically a recorded signal
-    divisor : array_like
-        Divisor data, typically an impulse response or filter that was
-        applied to the original signal
+    signal : 1-D array_like
+    Signal data, typically a recorded signal
+    divisor : 1-D array_like
+    Divisor data, typically an impulse response or filter that was
+    applied to the original signal
```

Comment:
Apart from the addition of 1-D, all docstring changes are not correct. Cf failing CI.